anxcloud/struct_dns_zones.go

```diff
@@ -41,3 +44,43 @@ func flattenDnsZones(dnsZones []zone.Zone) []interface{} {
 	}
 	return zones
 }
+
+func expandDNSServers(p []interface{}) []clouddns.DNSServer {
+	dnsServers := make([]clouddns.DNSServer, 0)
+	if len(p) < 1 {
+		return dnsServers
+	}
+
+	for _, elem := range p {
+		in := elem.(map[string]interface{})
+		dnsServer := clouddns.DNSServer{}
+
+		if v, ok := in["server"]; ok {
+			dnsServer.Server = v.(string)
+		}
+		if v, ok := in["alias"]; ok {
+			dnsServer.Alias = v.(string)
+		}
+
+		dnsServers = append(dnsServers, dnsServer)
+	}
+
+	return dnsServers
+}
+
+func flattenDNSServers(in []clouddns.DNSServer) []interface{} {
+	att := []interface{}{}
```

Comment:
please tell go the size this array will have, whenever you know the size beforehand

`att := make([]interface{}, 0, len(in))`